src/components/filters-header.tsx

```diff
@@ -41,6 +57,38 @@ export default function FiltersHeader({
       ? LOCATION_ROUTE
       : CATEGORY_TO_ROUTE_MAP[currentCategory]
   }${subCategory ? `/${subCategory}` : ""}`;
+
+  console.log("curernt category");
```

Comment:
Nitpick:
```suggestion
  console.log("current category");
```

But anyway, do you guys generally leave console logs in the code? It's not just something you put in for development but remove before merging?

Reply:
Yeah, some console logs are left in the code, though we are not supposed to leave them in the code.
I removed the logs.